.github/workflows/test_key_release_canary.yml

```diff
@@ -26,3 +26,22 @@ jobs:
       example-name: key_release
       locations: '["eastus2euap"]'
       manifest-modifiers: "KEY_RELEASE_IMAGE=http://confidentialsidecars.azurecr.io/skr:main"
+
+  key-release-grpc-canary-mcr:
+    name: Key Release GRPC Canary (MCR)
+    uses: ./.github/workflows/test_example.yml
+    secrets: inherit
+    with:
+      example-name: key_release_grpc
+      locations: '["eastus2euap"]'
+      manifest-modifiers: "KEY_RELEASE_IMAGE=http://mcr.microsoft.com/aci/skr:2.7"
```

Comment:
is there a reason for http over https?

Reply:
Honestly not sure, I've just been using it to be consistent with what @DomAyre has

Reply:
There's no specific reason, it's just how I initially did it. There is issue https://github.com/microsoft/confidential-aci-examples/issues/45 to address it, feel free to use HTTPS if you want for this example I'm not worried about it being consistent

Reply:
Since there is an issue to cover this comment, I'll mark it resolved and we can address the issue in a separate PR.